portfolio/src/main/webapp/script.js

```diff
@@ -196,4 +196,128 @@ function changeUsername() {
     document.getElementById('submit-comment-container').innerHTML =
         usernameResponse;
   });
-}
\ No newline at end of file
+}
+
+/** Create world map with city markers. */
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 25.246630, lng: 29.678410}, zoom: 1});
+
+  addCityMarker(
+      map, 40.768093, -73.981904, 'New York City',
+      'newyork');
+      
+  addCityMarker(
+      map, -33.824032, 151.187674, 'Sydney',
+      'sydney');
+
+  addCityMarker(
+      map, 14.563149, 121.036559, 'Manila',
+      'manila');
+
+  addCityMarker(
+      map, 34.105419, -117.706635, 'Los Angeles',
+      'losangeles');
```

Comment:
Define all of these as constants at the top.
I'd suggest creating an object to contain lat, lng and location name, and then having a list or a map of those
Access would then be something like `varName[CITY].lat`  .....